woocommerce-services.php

```diff
@@ -703,6 +699,11 @@ protected function add_method_to_shipping_zone( $zone_id, $method_id ) {
 		}
 
 		public function init_core_wizard_shipping_config() {
+			// @todo Verify what happens if WCS&T was installed without this, and WC Shipping is then deactivated.
```

Comment:
We already have a notice in place which nags in case WC Shipping is active and WCS&T is active as well.
We can adjust to tell them to update the WCS&T (to the version that can co-exist with WC Shipping) or remove it to be able to use WC Shipping.